forum/views.py

```diff
@@ -38,6 +43,8 @@ def get_forum(self):
 class PostDetailView(DetailView):
     model = Post
     pk_url_kwarg = 'post_id'
+    # how do i access passed in parameter?
+    print(pk_url_kwarg)
```

Comment:
Remove this. Why do you need to access a URL kwarg?

Reply:
I wanted to access data for that specific instance of post, but it is probably not necessary in the view file in real life.  the html file has access to the object, and admin will allow you to make any change you need.

Reply:
removed comment